### ISSUE TITLE
counter example: reflex (0.1.34) run failed for the first version of reflex counter

### ISSUE THREAD

**Issue by milochen0418** [pynecone-io/pynecone-examples#122]
```
Traceback (most recent call last):
  File "/Users/milochen/anaconda3/envs/rx/bin/rf", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/Users/milochen/anaconda3/envs/rx/lib/python3.11/site-packages/typer/main.py", line 214, in __call__
    return get_command(self)(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/milochen/anaconda3/envs/rx/lib/python3.11/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/milochen/anaconda3/envs/rx/lib/python3.11/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
         ^^^^^^^^^^^^^^^^
  File "/Users/milochen/anaconda3/envs/rx/lib/python3.11/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/milochen/anaconda3/envs/rx/lib/python3.11/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/milochen/anaconda3/envs/rx/lib/python3.11/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/milochen/anaconda3/envs/rx/lib/python3.11/site-packages/typer/main.py", line 532, in wrapper
    return callback(**use_params)  # type: ignore
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/milochen/anaconda3/envs/rx/lib/python3.11/site-packages/reflex/rf.py", line 126, in run
    app = prerequisites.get_app()
          ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/milochen/anaconda3/envs/rx/lib/python3.11/site-packages/reflex/utils/prerequisites.py", line 105, in get_app
    return __import__(module, fromlist=(constants.APP_VAR,))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ModuleNotFoundError: No module named '.'
```

**Comment by masenf** [comment]
@milochen0418 can you test on `reflex ~= 0.2.0a1`, which is the latest candidate

**Comment by milochen0418** [comment]
> @milochen0418 can you test on `reflex ~= 0.2.0a1`, which is the latest candidate

@masenf 
Nice ~ after testing, this issue is solved in `reflex==0.2.0a1` .
<img width="1440" alt="Screenshot 2023-06-30 at 2 32 09 PM" src="https://github.com/pynecone-io/pynecone-examples/assets/12568287/43d09cea-306f-4c57-b829-cc5379ae2b78">

**Comment by milochen0418** [comment]
It is solved in reflex==0.2.0
